docs/reports/mapped_deprecated_doid.md

```diff
@@ -2,6 +2,7 @@
 [Interactive FlatGithub table](https://flatgithub.com/monarch-initiative/mondo-ingest?filename=src/ontology/reports/doid_mapped_deprecated_terms.robot.template.tsv)
 
 ### Mapped deprecated terms
-| mondo_id   | source_id            | source             |
-|:-----------|:---------------------|:-------------------|
-| ID         | A oboInOwl:hasDbXref | >A oboInOwl:source |
\ No newline at end of file
+| mondo_id      | source_id            | source                   |
+|:--------------|:---------------------|:-------------------------|
+| ID            | A oboInOwl:hasDbXref | >A oboInOwl:source       |
+| MONDO:0011893 | DOID:0110578         | MONDO:equivalentObsolete |
```

Comment:
Doesn't have to do with synonym sync, but just observing that `n` for "mapped deprecated - DOID" went from 0 to 1.